app/models/experiment.js

```diff
@@ -3,14 +3,16 @@ Manage data about one or more documents in the experiments collection
  */
 
 import DS from 'ember-data';
+import JamModel from '../mixins/jam-model';
 
-export default DS.Model.extend({
-  title: DS.attr('string'),
-  description: DS.attr('string'),
-  structure: DS.attr(),  // TODO: Nested document
-  permissions: DS.attr(),
+export default DS.Model.extend(JamModel, {
+    title: DS.attr('string'),
+    description: DS.attr('string'),
+    active: DS.attr('boolean'),
+    structure: DS.attr(),
+    permissions: DS.attr(),
 
-  administrators: DS.hasMany('administrator'),  // TODO: Is Many-to-many correct relationship type?
-  history: DS.hasMany('history'),
-  sessions: DS.hasMany('session'),
+    administrators: DS.hasMany('admin'),
```

Comment:
Can you explain how this relationship is expected to work? Semantically I read: "All admins associated with this experiment". 

Is this implicitly the list of all admins?